content/02.main-text.md

```diff
@@ -257,3 +263,25 @@ Manubot provides an ideal platform for perpetual reviews [@arxiv:1502.01329; @ta
 
 Our process represents an early step toward open massively collaborative reviews, and there are certainly aspects that can be improved.
 We invite the scientific community to adapt and build upon our experience and open software.
+
+### The open science movement
```

Comment:
This section could be cut down more... what do people think should go or stay?

Reply:
The open science discussion works much better here than in the intro.  I made specific comments on what could be revised.  Overall, I think the first paragraph could be cut substantially (or removed with essential references integrated elsewhere).  The second and third paragraphs fit well.